backend/api/appsettings.json

```diff
@@ -2,8 +2,8 @@
   "BaseUrl": "/api",
   "HealthChecks": {
     "Port": "8080",
-    "LivePath": "/api/health/live",
-    "ReadyPath": "/api/health/ready"
+    "LivePath": "/health/live",
```

Comment:
This is to fix the local environment.  I'm not certain if this will break DEV.  If so I will have to modify the config in OpenShift.